### DISCUSSION TITLE
What is the PnP equivalent of Get-SPOSite -Template REDIRECTSITE#0?

### DISCUSSION THREAD

**Discussion by haneef95** [pnp/powershell#1300] [Q&A]
Hi,

I would like to know the PnP equivalent of this command:
`Get-SPOSite -Template REDIRECTSITE#0`

Follow up from: https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3018

Thanks,

**Comment by veronicageek** [comment]
@haneef95 - Try `Get-PnPTenantSite | Where-Object { $_.Template -eq "REDIRECTSITE#0" }`

**Reply by haneef95** [reply]
Thanks, will give that a shot. But, that's client side filtering right? So, it's downloading the whole list, before only displaying a portion of it?

**Reply by haneef95** [reply]
This worked well @veronicageek.

Thanks
https://github.com/pnp/powershell/discussions/1300#discussioncomment-1560604

**Comment by AndersRask** [comment]
Where-Object is taking the object from the pipeline of the Get-PnPTenantSite, so yes, it will fetch all sites first.

You should be able to do it like this:

Get-PnPTenantSite -Template "REDIRECTSITE#0"

For the same result, but faster if you have many site collections

Fra: Haneef ***@***.***>
Sendt: Friday, 29 October 2021 11.27
Til: pnp/powershell ***@***.***>
Cc: Subscribed ***@***.***>
Emne: Re: [pnp/powershell] What is the PnP equivalent of Get-SPOSite -Template REDIRECTSITE#0? (Discussion #1300)


Thanks, will give that a shot. But, that's client side filtering right? So, it's downloading the whole list, before only displaying a portion of it?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/pnp/powershell/discussions/1300#discussioncomment-1557852>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AAMTDTUY47K2U53KMYZU4T3UJJZILANCNFSM5GYF7D4Q>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by haneef95** [reply]
Thanks @AndersRask , that worked well.

And of course, it'll be faster as the filtering is done on the server-side before delivering the results to the client, which is preferable.

However, a slight weird issue though, it would only work with:
`Get-PnPTenantSite -Template "RedirectSite#0"`

I guess, that parameter is case sensitive. If you could please edit your answer and notify me, I'll mark it as the answer.

Thanks,

**Reply by haneef95** [reply]
Also, would you happen to know how I could find which Site/URL it's redirecting to?

Thanks
